js/apps/system/_admin/aardvark/APP/frontend/js/views/applicationDetailView.js

```diff
@@ -192,7 +192,7 @@
             } else if (result) {
               info = (
                 '<p>Script results:</p>' +
-                '<pre>' + JSON.stringify(result, null, 2) + '</pre>'
+                '<pre>' + arangoHelper.escapeHtml(result) + '</pre>'
```

Comment:
Note that moving the `JSON.stringify` into `escapeHtml` will result in string results being displayed differently than before:

Previously:
```
# script output
module.export = "success"

# generated html
<pre>"success"</pre>
```

Now:
```
# script output
module.export = "success"

# generated html
<pre>success</pre>
```

This results in some ambiguity if the string output looks like JSON, e.g.

```
module.exports = "[]"

# now looks the same as

module.exports = []
```

Reply:
I agree this is a change, but do not really think this is an actual issue.